app/views/smart_answers/custom_result.html.erb

```diff
@@ -26,10 +26,14 @@
       "action": "complete",
       "tool_name": @presenter.title,
     }.to_json %>">
-    <%= render "govuk_publishing_components/components/title", {
-      title: outcome.title,
-      margin_bottom: 6,
-    } %>
+    <div class="govuk-!-margin-top-8">
```

Comment:
I've avoided doing this extra top margin on the ones I've done so far as there's a different way of handling this using classes on the `main` element now. If you have a look at this section in the design system, it explains which classes to use: https://design-system.service.gov.uk/styles/layout/#add-vertical-space